### PR TITLE
maximmax42.CustomRP version 1.11.6

### PR DESCRIPTION
**Pull request by Boxersteavee**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26644)

### CODE REVIEW

File: manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: maximmax42.CustomRP
+PackageVersion: 1.11.6
+PackageLocale: en-US
+Publisher: maximmax42
+PackageName: CustomRP
+License: Copyright © 2018-2021 maximmax42
```
Comment (quhxl) [inline manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml:9]:
```suggestion
License: MIT
LicenseUrl: https://github.com/maximmax42/Discord-CustomRP/blob/master/LICENSE
Copyright:  Copyright © 2018-2021 maximmax42
```

**Review by quhxl — CHANGES_REQUESTED** [review]
Nothing wrong with your manifest. Just added some more metadata to avoid creating an extra PR for this.

File: manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: maximmax42.CustomRP
+PackageVersion: 1.11.6
+PackageLocale: en-US
+Publisher: maximmax42
```
Comment (quhxl) [inline manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml:7]:
```suggestion
Publisher: maximmax42
PublisherSupportUrl: https://github.com/maximmax42/Discord-CustomRP/issues
```

File: manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: maximmax42.CustomRP
+PackageVersion: 1.11.6
+PackageLocale: en-US
+Publisher: maximmax42
+PackageName: CustomRP
```
Comment (quhxl) [inline manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml:8]:
```suggestion
PackageName: CustomRP
PackageUrl: https://www.customrp.xyz/
```

**Review by quhxl — CHANGES_REQUESTED** [review]

File: manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: maximmax42.CustomRP
+PackageVersion: 1.11.6
+PackageLocale: en-US
+Publisher: maximmax42
+PackageName: CustomRP
+License: Copyright © 2018-2021 maximmax42
+ShortDescription: CustomRP Setup
```
Comment (quhxl) [inline manifests/m/maximmax42/CustomRP/1.11.6/maximmax42.CustomRP.locale.en-US.yaml:10]:
```suggestion
ShortDescription: CustomRP is a Custom Discord Rich Presence Manager.

```

**Review by quhxl — CHANGES_REQUESTED** [review]

**Comment by Boxersteavee** [comment]
> Nothing wrong with your manifest. Just added some more metadata to avoid creating an extra PR for this.

I think I sorted it. First time doing this but glad you managed to help me sort it!

**Comment by Boxersteavee** [comment]
Also thought I should add:
This manifest an pull request were made using the Winget create tool

**Review by ImJoakim — APPROVED** [review]
image

**Pull request merged**